build/compile/LzmaArchive.targets

```diff
@@ -115,11 +115,7 @@
       </GenerateNuGetPackagesArchiveVersion>
       <PropertyGroup>
         <NugetPackagesArchiveName>nuGetPackagesArchive.$(NuGetPackagesArchiveVersion).lzma</NugetPackagesArchiveName>
```

Comment:
Yes, just leave these alone.  There is no value in trying to refactor this.  The `$(NuGetPackagesArchiveVersion)` isn't a static property - it changes depending on which `<PackageReference>` items are in the `dotnet new` templates.  So yes, you'd have to build the product first, before you could generate this `$(NuGetPackagesArchiveVersion)`.